wp-content/themes/eighshift-task/src/blocks/layout/header/_header-navigation.scss

```diff
@@ -0,0 +1,35 @@
+$header-navigation: (
+  header-nav-spacing: $base-gutter-size,
+  header-nav-negative-spacing: -$base-gutter-size,
+  header-nav-text: $base-text-color,
+  header-nav-highlight-bg: $base-primary-color,
+  header-nav-highlight-text: $base-white-color,
+);
+
+.header-navigation {
+  @extend %list-reset;
+  display: flex;
+  align-items: center;
+  margin: 0 map-get($header-navigation, 'header-nav-negative-spacing');
+
+  &__item {
+    padding-left: map-get($header-navigation, 'header-nav-spacing');
+    padding-right: map-get($header-navigation, 'header-nav-spacing');
+
+    &--menu-highlight {
+      .header-navigation__link {
+        background-color: map-get($header-navigation, 'header-nav-highlight-bg');
+        color: map-get($header-navigation, 'header-nav-highlight-text');
+        padding: 14px 30px;
+        border-radius: 500px;
```

Comment:
add 100% or something like this.

Reply:
To achieve the style of button from the design I would have to set the value of `border-radius` to the half the height of the element. 50% will not work. Easier solution to this is just set the value to some high number. 

https://stackoverflow.com/questions/27233295/possible-to-make-border-radius-equal-to-half-the-total-height-without-javascript

Reply:
👍